OmiseSwift/Destroyable.swift

```diff
@@ -5,20 +5,20 @@ public protocol Destroyable { }
 public extension Destroyable where Self: ResourceObject {
     public typealias DestroyOperation = Operation<Self>
     
-    public static func destroyOperation(parent: ResourceObject?) -> DestroyOperation {
+    public static func destroyOperation(_ parent: ResourceObject?, id: String) -> DestroyOperation {
```

Comment:
I should close my pr or not because it changed in this pr
https://github.com/chakrit/omise-swift/pull/23